src/ddsrt/cmake/events.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright(c) 2020 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+#include "dds/ddsrt/events.h"
+
+#if defined __APPLE__
```

Comment:
Remember to check for `__FreeBSD__` here as well.

Reply:
Already on it.

Reply:
This is now implemented.